homeassistant/components/honeywell/climate.py

```diff
@@ -8,6 +8,7 @@
 from aiohttp import ClientConnectionError
 from aiosomecomfort import (
     AuthError,
+    ConnectionError as asc_ConnectionError,
```

Comment:
Mixing snake_case and PascalCase is unexpected. Just make it PascalCase.

Reply:
https://github.com/home-assistant/core/pull/104649